components/Shortcuts/shortcut_sequence.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {MessageDescriptor} from 'react-intl';
+
+type Props = {
+    shortcut: MessageDescriptor;
+};
+
+export default function ShortcutSequence({shortcut}: Props) {
+    if (!shortcut?.defaultMessage) {
+        return null;
+    }
+
+    const shortCut = shortcut.defaultMessage.split('\t');
+    let keys = null;
+    if (shortCut.length > 1) {
+        keys = shortCut[1].split('|').map((key) => (
+            <span
+                className='shortcut-key'
+                key={key}
+            >
+                {key}
+            </span>
+        ));
+    }
```

Comment:
> I have implemented the sidebar team tooltip with a new look, but have some issue regarding the how to pass the order?

Here, instead of reading `defaultMessage` directly, you can pass `shortcut` (a `MessageDesciptor`) to `formatMessage()` from 'react-intl' e.g.
```tsx
const {formatMessage} = useIntl();
const formatted = formatMessage(shortcut, values);
```

`values` should be a new prop (that replaces `order` in `team_button.tsx`) which is an object with key/values that replaces the placeholders. e.g. as it was before

```tsx
const toolTipHelp = formatMessage({
    id: 'team.button.tooltip',
    defaultMessage: 'Ctrl+Alt+{order}',
},
{
    order: this.props.order,
});
```

In this case, `formatMessage` is just moving from `team_button.tsx` to `shortcut_sequence.tsx`

Reply:
@calebroseland actually I have changed that login and moved the shortcut to all shorts object, here is the updated code https://github.com/mattermost/mattermost-webapp/blob/26e52e9c2ae3cd88d9e0bb1fecc24dac9e4800db/components/legacy_team_sidebar/components/team_button.tsx, if it is possible by this method I will follow this or I will change as you previously mentioned.